ooniprobe/Engine/Engine.h

```diff
@@ -15,6 +15,8 @@
 
 + (NSString*) versionMK;
 
++ (NSString*) getUUID;
```

Comment:
same comment as for Android: this is not a cached UUID, it's new each time. So you need to use another less ambiguous name here, like `newUUID4` and also cache it yourself.